packages/terra-form-select/src/ComboboxField.jsx

```diff
@@ -99,6 +99,11 @@ const propTypes = {
    * The value of the select. Can be a string, number, or array of strings/numbers.
    */
   value: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.array]),
+  /**
+   * @private
+   * The id of the input field.
+   */
+  inputId: PropTypes.string,
```

Comment:
`inputId` shouldn't be a prop on the Field components, only on the corresponding variants. The id of the Field components should map into the inputId of the corresponding variants.

Reply:
Resolved in f1d8d8f